ush/config_defaults.yaml

```
@@ -2892,7 +2892,7 @@ smoke_dust_parm:
   # Flag turning on/off Smoke and Dust
   #
   # EBB_DCYCLE:
-  # 1: for retro, 2: for forecast
+  # 1: for retro, 2: for forecast (considered experimental in the SRW)
```

SRW App (similar to the suggested change from @WeirAE in PR #1250)

```suggestion
  # 1: for retro, 2: for forecast (considered experimental in the SRW App)
```

---

Makes sense! Done and resolving